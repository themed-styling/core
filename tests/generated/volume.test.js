import 'jest-styled-components'

import { volume } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(volume, 'volume')
