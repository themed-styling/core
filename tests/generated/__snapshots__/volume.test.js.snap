// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`volume renders 0 1`] = `
Array [
  .c0 {
  volume: 0;
}

<div
    className="c0"
  />,
  .c0 {
  volume: 0;
}

<div
    className="c0"
  />,
  .c0 {
  volume: 0;
}

<div
    className="c0"
  />,
  .c0 {
  volume: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`volume renders function 1`] = `
Array [
  .c0 {
  volume: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  volume: has;
}

<div
    className="c0"
  />,
  .c0 {
  volume: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  volume: has;
}

<div
    className="c0"
  />,
]
`;

exports[`volume renders invalid theme path 1`] = `
Array [
  .c0 {
  volume: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  volume: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  volume: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  volume: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`volume renders negative number 1`] = `
Array [
  .c0 {
  volume: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  volume: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  volume: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  volume: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`volume renders object [breakpoints + standard] 1`] = `
Array [
  .c0 {
  volume: July;
}

@media screen and (min-width:November) {
  .c0 {
    volume: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    volume: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    volume: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    volume: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  volume: July;
}

@media screen and (min-width:November) {
  .c0 {
    volume: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    volume: September;
  }
}

@media screen and (min-width:November) {
  .c0 {
    volume: November;
  }
}

@media screen and (min-width:August) {
  .c0 {
    volume: August;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  .c0 {
  volume: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    volume: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    volume: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    volume: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    volume: June;
  }
}

<div
    className="c0"
  />,
  .c0 {
  volume: January;
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    volume: December;
  }
}

@media screen and (min-width:November) {
  .c0 {
    volume: April;
  }
}

@media screen and (min-width:November) {
  .c0 {
    volume: May;
  }
}

@media screen and (min-width:August) {
  .c0 {
    volume: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`volume renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    volume: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    volume: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    volume: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    volume: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    volume: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    volume: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    volume: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    volume: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    volume: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    volume: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    volume: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    volume: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    volume: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    volume: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    volume: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    volume: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`volume renders object [mediaQueries + standart] 1`] = `
Array [
  .c0 {
  volume: July;
}

@media May {
  .c0 {
    volume: August;
  }
}

@media April {
  .c0 {
    volume: September;
  }
}

@media January {
  .c0 {
    volume: November;
  }
}

@media April {
  .c0 {
    volume: August;
  }
}

@media June {
  .c0 {
    volume: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  volume: July;
}

@media May {
  .c0 {
    volume: August;
  }
}

@media April {
  .c0 {
    volume: September;
  }
}

@media January {
  .c0 {
    volume: November;
  }
}

@media April {
  .c0 {
    volume: August;
  }
}

@media June {
  .c0 {
    volume: November;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  .c0 {
  volume: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    volume: November;
  }
}

@media April {
  .c0 {
    volume: January;
  }
}

@media January {
  .c0 {
    volume: December;
  }
}

@media April {
  .c0 {
    volume: April;
  }
}

@media June {
  .c0 {
    volume: May;
  }
}

<div
    className="c0"
  />,
  .c0 {
  volume: April;
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    volume: November;
  }
}

@media April {
  .c0 {
    volume: January;
  }
}

@media January {
  .c0 {
    volume: December;
  }
}

@media April {
  .c0 {
    volume: April;
  }
}

@media June {
  .c0 {
    volume: May;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`volume renders object [mediaQueries] 1`] = `
Array [
  @media May {
  .c0 {
    volume: July;
  }
}

@media April {
  .c0 {
    volume: August;
  }
}

@media January {
  .c0 {
    volume: September;
  }
}

@media April {
  .c0 {
    volume: November;
  }
}

@media June {
  .c0 {
    volume: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    volume: July;
  }
}

@media April {
  .c0 {
    volume: August;
  }
}

@media January {
  .c0 {
    volume: September;
  }
}

@media April {
  .c0 {
    volume: November;
  }
}

@media June {
  .c0 {
    volume: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    volume: January;
  }
}

@media April {
  .c0 {
    volume: December;
  }
}

@media January {
  .c0 {
    volume: April;
  }
}

@media April {
  .c0 {
    volume: May;
  }
}

@media June {
  .c0 {
    volume: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    volume: January;
  }
}

@media April {
  .c0 {
    volume: December;
  }
}

@media January {
  .c0 {
    volume: April;
  }
}

@media April {
  .c0 {
    volume: May;
  }
}

@media June {
  .c0 {
    volume: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`volume renders postitive number 1`] = `
Array [
  .c0 {
  volume: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  volume: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  volume: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  volume: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`volume renders string 1`] = `
Array [
  .c0 {
  volume: July;
}

<div
    className="c0"
  />,
  .c0 {
  volume: July;
}

<div
    className="c0"
  />,
  .c0 {
  volume: June;
}

<div
    className="c0"
  />,
  .c0 {
  volume: June;
}

<div
    className="c0"
  />,
]
`;

exports[`volume renders undefined 1`] = `
Array [
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
  <div
    className=""
  />,
]
`;

exports[`volume renders valid theme path 1`] = `
Array [
  .c0 {
  volume: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  volume: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  volume: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  volume: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
